classes/02class/exercises/c02-network08/mail2kvanitha/README.md

```diff
@@ -0,0 +1,65 @@
+# c02-network08
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    Inbound Rules are for traffic incoming to the subnet to which the NACLs are attached. 
+    Outbound Rules are for the traffic leaving the subnet to which the NACLs are attached. i
+    By default, the default NACLs  allow ALL inbound and outbound traffic. 
+    But for custom NACLs(devopsacademy-nacls),by default ALL inbound and outbound traffic is denied till RULES are added.
+    ```
+    
+  - What is going to happen by default? Everything allowed or denied?
+    ```
+    By default, the default NACLs allow ALL inbound and outbound traffic to/from the subnet to which it is attached. 
```

Comment:
the question is related to the table in the exercise description.

Reply:
thanks, got it now. Updated the answer accordingly.